runtime/gc_modron_startup/mmparseXgc.cpp

```diff
@@ -1447,6 +1447,26 @@ gcParseXgcArguments(J9JavaVM *vm, char *optArg)
 			continue;
 		}
 
+		/* Check if there is a request a suballocator reservation increment size. */
+		if (try_scan(&scan_start, "suballocatorIncrementSize=")) {
+			if (!scan_udata_memory_size_helper(vm, &scan_start, &extensions->suballocatorIncrementSize, "suballocatorIncrementSize=")) {
+				returnValue = JNI_EINVAL;
+				break;
+			}
+			if (0 == extensions->suballocatorIncrementSize) {
+				j9nls_printf(PORTLIB, J9NLS_ERROR, J9NLS_GC_OPTIONS_VALUE_MUST_BE_ABOVE, "-Xgc:suballocatorIncrementSize=", (UDATA)0);
+				returnValue = JNI_EINVAL;
+				break;
+			}
+			continue;
+		}
+
+		/* Check if there is a request to disable the mmap-based allocation for the suballocator (linux only). */
+		if (try_scan(&scan_start, "suballocatorQuickAllocDisable")) {
+			extensions->suballocatorQuickAlloc = false;
+			continue;
+		}
+
```

Comment:
Please add `suballocatorQuickAllocEnable` option as well for symmetry.

Reply:
I've added it. But in the OMR patch: https://github.com/eclipse/omr/blob/6a68a292908d30fb6552a46d585d87af33f21904/port/common/omrportcontrol.c#L93-L103 
I've left handling for this case as a no-op since `true` is the default for `suballocatorQuickAlloc`. Let me know if you think I should add in case in `omrport_control` anyways.

Reply:
`omrport_control(OMRPORT_CTLDATA_ALLOCATE32_QUICK_ALLOC, extensions->suballocatorQuickAlloc);` call from Memory Manager should set key value regardless it's default value. What prevents just to do `PPG_mem_mem32_subAllocHeapMem32.suballocator_quickAlloc = value;`?

Reply:
Nothing prevents it. I was just trying to follow similar logic to what was provided with the other options.

I can change it to that.

Reply:
May be (I am not insisting) because of `value` parameter has `uintptr_t` type it is better to use this type explicitly at both sides instead of `bool`/`BOOLEAN`? In GC for  `extensions->suballocatorQuickAlloc` and in Port for `PPG_mem_mem32_subAllocHeapMem32.suballocator_quickAlloc`, just to prevent types conversion?

Reply:
I like that idea too.